docs(t&s): update procedures (#4826)
scaleway/docs-content: pages/dedibox-account/how-to/disable-two-factor-authentication.mdx

## How to restore access to your account

If you lose both your MFA app and backup codes, the only way to regain access to your account is through a manual identity verification.

> To do so, contact our Trust and Safety team with the following information:

Reviewer: ```suggestion
To do so, make a request to our Trust and Safety team. You must make your request from the e-mail address registered on your account. In your email, provide the following information:
```

> * Make your request from the e-mail address registered on your account

Reviewer: ```suggestion
```

> * Indicate your Username

Reviewer: ```suggestion
* Your username
```

> To do so, make a request to our Trust and Safety team. You must make your request from the e-mail address registered on your account. In your email, provide the following information:

Reviewer 2: ```suggestion
To do so, make a request to our Trust and Safety team. You must make your request from the email address registered on your account. In your email, provide the following information:
```

> * Provides us the telephone number registered on your account

Reviewer: ```suggestion
* The telephone number registered on your account
```

> * If you have a individual account, your passport or local ID card

Reviewer 2: ```suggestion
* If you have an individual account, your passport or local ID card
```

> * If you have a company account, the certificate of incorporation and a copy of the last Dedibox invoice you received

Reviewer 2: ```suggestion
* If you have a corporate account, the certificate of incorporation and a copy of the last Dedibox invoice you received
```

* If you have a company account, the certificate of incorporation and a copy of the last Dedibox invoice you received

> * Submit the complete set of required documents via email to [mailto:document@scaleway.com](document@scaleway.com)

Reviewer: ```suggestion

Submit the complete set of required documents via email to [mailto:document@scaleway.com](document@scaleway.com)
```
